tailwind.config.js

```diff
@@ -5,7 +5,13 @@ module.exports = {
   "./src/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-  extend: {},
+    extend: {
+      fontFamily: {
+        header: ['Fjalla One', 'sans-serif'],
+        body: ['Merriweather Sans', 'sans-serif'],
+        button: ['Merriweather Sans', 'sans-serif'],
+      },
+    },
```

Comment:
request(nb): Would you mind taking the fonts work out and doing it in another PR? I think this PR may be a lot, and I don't want to hold up other work.